app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -559,6 +567,10 @@ def rbac_user_delete_restriction?(user)
     ["admin", session[:userid]].include?(user.userid)
   end
 
+  def rbac_user_copy_restriction?(user)
+    user.userid == "admin"
```

Comment:
@romanblanco isn't better to check it by `super_admin_user?` I guess that `EvmRole-super_administrator` can only one